code/modules/lighting/lighting_system.dm

```diff
@@ -72,7 +73,9 @@
 		return
 
 	radius = max(LIGHTING_MIN_RADIUS, new_luminosity)
+	radius = owner.get_light_range(radius)
```

Comment:
Well you could save on var access overhead with `radius = owner.get_light_range(max(LIGHTING_MIN_RADIUS, new_luminosity))` but I'm not sure it matters a lot.